cf_units/__init__.py

```diff
@@ -1210,6 +1209,35 @@ def is_time_reference(self):
         """
         return self.calendar is not None
 
+    def is_long_time_interval(self):
+        """
+        Defines whether this unit describes a time unit with a long time
+        interval ("months" or "years"). These long time intervals *are*
+        supported by `UDUNITS2` by are not supported by `netcdftime`. This
```

Comment:
typo: These long time intervals are supported by `UDUNITS2` **but** are not supported by `netcdftime`.